docs/03-declarative-layout.md

```diff
@@ -29,7 +28,116 @@ configurable rules.
 
 @DOC_layout_WIDGET_LIST@
 
-## Placing widgets
+### The different type of widget boxes (Wibox)
+
+Awesome API the word "wibox" to describe an area of the screen filled with
+widgets. There is many subvariants of wiboxes with specialized role such as
+widget bars or tooltips. All variants mostly share the same characteristics,
+but add some extra features to make those specialized widget boxes easier to
+work with.
+
+@DOC_awful_popup_wiboxtypes_EXAMPLE@
+
+The normal `wibox` is the base class for each of these type. It is extremely
+flexible and allows to place just about anything on the screen. However it
+requires a lot of repetitive boiler plate code to use directly. For example,
+the user need to compute the optimal size by hand or use `awful.placement`.
+
+The `awful.wibar` specialization allows to attach a `wibox` to a screen edge
+and prevent clients from using this area when tiled.
+
+The `awful.popup` allows to easily place widgets on the screen. It automatically
+resize itself to fit the optimal widget size. It also have helper properties
+and method to make it easy to place it on the screen. It both absolute
```

Comment:
it both _supports_ absolute

(Also, I wouldn't use `both` on a list with three entries, how about `It supports all of` or just `It supports absolute, relative and manual positioning`?)